### DISCUSSION TITLE
Writing to a syslog unix socket

### DISCUSSION THREAD

**Discussion by peterand-pa** [dmachard/DNS-collector#591] [Q&A] [answered]
How should writing to a unix socket be configured ?
I'm running syslog-ng listening to a unix file socket.

I put in go-dnscollector config
    - name: localsyslog
      syslog:
        transport: unix
        remote-address: "/var/tmp/dnstap.sock"

I'm currently getting this error :
INFO: 2024/02/02 17:27:01.125579 [localsyslog] logger=syslog - connecting to unix:///var/tmp/dnstap.sock ...
ERROR: 2024/02/02 17:27:01.125748 [localsyslog] logger=syslog - dial unix /var/tmp/dnstap.sock: connect: protocol wrong type for socket

**Comment by dmachard** [comment]
I need to do some tests but it should be possible.
Can you open issue to track that ? Thank for reporting

**Reply by dmachard** [reply, marked as answer]
fixed in release v0.41.0
